Gem/Code/Source/MSAA_RPI_ExampleComponent.cpp

```diff
@@ -101,6 +101,8 @@ namespace AtomSampleViewer
             SampleComponentManagerRequestBus::Broadcast(&SampleComponentManagerRequests::SetNumMSAASamples, m_numSamples);
             SampleComponentManagerRequestBus::Broadcast(&SampleComponentManagerRequests::ResetRPIScene);
 
+            m_scene = AZ::RPI::RPISystemInterface::Get()->GetSceneByName(AZ::Name("RPI"));
+
```

Comment:
This should no longer be necessary. I fixed ResetScene() to update the scene directly a week or so ago. https://github.com/o3de/o3de-atom-sampleviewer/pull/488/

Reply:
I haven't updated the PR since last week. I'll close it.